avm/res/cache/redis/main.bicep

```diff
@@ -267,7 +267,7 @@ module redis_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.4.0'
           }
         }
       ]
-      manualPrivateLinkServiceConnections: privateEndpoint.?manualPrivateLinkServiceConnections == true
+      manualPrivateLinkServiceConnections: privateEndpoint.?isManualConnection == true
```

Comment:
I see that in quite a few files the `privateLinkServiceConnections` is incorrect. Per PE deployment there should be 2 checks for `isManualConnection` while some files have only one. I'll comment on each.

To fix, the privateLinkServiceConnections should be configured as
`privateLinkServiceConnections: privateEndpoint.?isManualConnection != true`

Reply:
I re-created the Private Endpoints for all modules in this PR using the original specs, I think now everything should be fine.